src/components/Headings.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import clsx from 'clsx'
+
+export const H1 = ({ children, className }) => {
+  const defaultClass =
+    'max-w-4xl font-display text-5xl font-medium tracking-tight text-slate-900 sm:text-7xl'
+  return <h1 className={clsx(defaultClass, className)}>{children}</h1>
+}
+
+export const H2 = ({ children, className, id }) => {
+  const defaultClass = 'font-display text-3xl tracking-tight sm:text-4xl'
+  return (
+    <h2 id={id} className={clsx(defaultClass, className)}>
+      {children}
+    </h2>
+  )
```

Comment:
shouldn't there be option to pass all of the attributes of H1, possibly through React.forwardRef!

Reply:
Thanks for the suggestion. I utilized React.forwardRef for H1 and H2, and removed the id props from the H2 component.

I'm not sure why deployment has failed. When I went to the detail of "Deployment has failed", the Vercel page asked me to log in. It would be great if you have a look at this issue.

Reply:
I just needed to add display names to each component to resolve a linting issue.